ssn/integrated/ssnx.ttl

```diff
@@ -54,7 +54,30 @@ time:TemporalEntity a owl:Class .
   owl:imports sosa: , ssn: .
 
 
+## Features of interest and observable property
 
+oldssn:FeatureOfInterest a owl:Class ;
+  rdfs:label "Feature of Interest"@en ;
+  skos:definition """A feature is an abstraction of real world phenomena (thing, person, event, etc)."""@en ;
+  dcterms:source """skos:exactMatch 'feature' [O&M]
+    http://www.opengeospatial.org/standards/om""" ;
+  rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" ;
+  owl:deprecated true ;
+  owl:equivalentClass sosa:FeatureOfInterest ;
+  rdfs:isDefinedBy <http://purl.oclc.org/NET/ssnx/ssn> .
```

Comment:
followed the integration methodology for oldssn:FeatureOfInterest, is equivalent to sosa:FeatureOfInterest